docs-content/getting-started/5_backend-logging/04_ruby/1_overview.md

```diff
@@ -17,7 +17,4 @@ If you don't see one of your languages / frameworks below, reach out to us in ou
     <DocsCard title="Other Ruby Frameworks" href="./other.md">
         {"Integrate logging in other Ruby Frameworks."}
     </DocsCard>
-    <DocsCard title="Ruby OpenTelemetry" href="../../7_native-opentelemetry/3_logging.md">
-        {"Integrate logging with Native OpenTelemetry."}
-    </DocsCard>
```

Comment:
@Vadman97 these docs linked to a page that showed JS example code. Was thinking it's better to hide these until we have some Ruby code examples for this. Is that cool with you?

Reply:
yeah, sounds great to me